### ISSUE TITLE
[Tutorial] Possible improvement in the "Clean labels using your model loss" tutorial

### ISSUE THREAD

**Issue by leireropl** [recognai/rubrix#698]
After running the function for computing the loss in this tutorial, it is required to turn the dataset into a Pandas data frame. When running the code, the following warnings are shown:

**/var/folders/ym/p8klf4tx3cb_7q9vsngj0z4w0000gn/T/ipykernel_16926/2192143421.py:4: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).**
**input_ids = torch.tensor(batch["input_ids"], device=device)/var/folders/ym/p8klf4tx3cb_7q9vsngj0z4w0000gn/T/ipykernel_16926/2192143421.py:5: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).**
 **attention_mask = torch.tensor(batch["attention_mask"], device=device)
/var/folders/ym/p8klf4tx3cb_7q9vsngj0z4w0000gn/T/ipykernel_16926/2192143421.py:6: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  labels = torch.tensor(batch["labels"], device=device)**

Therefore, these variables can be changed to:
    **input_ids = batch["input_ids"].clone().detach()**
    **attention_mask = batch["attention_mask"].clone().detach()**
    **labels = batch["labels"].clone().detach()**

Then, the warning disappears.

**Comment by dcfidalgo** [comment]
Done in #714
